alert-database/src/main/resources/liquibase/6.0.0/provider-fields.xml

```diff
@@ -103,7 +103,7 @@
                     WHERE type_id IN (
                         SELECT id
                         FROM alert.descriptor_types
-                        WHERE type = 'CHANNEL'
+                        WHERE type = 'PROVIDER'
```

Comment:
This is the actual fix for the 3 upgrade issues that were found.  The config name was being added the channel distribution configurations and not the provider distribution configuration.

Reply:
Ugh. How did I mess that up?